app/views/users/logout.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= _('Are you sure you want to logout?') %></h2>
+<%= button_to _('yes'), logout_users_url, :class => 'btn btn-primary' %>
```

Comment:
The reason this page is being added is that if you access the URL directly (e.g. bookmarked log out link from menu) you'd get 404. There's no way to get to this page from Foreman navigation, so there's no "previous" page. I could add "No" link that would always point to let's say dashboard or hosts index...